### ISSUE TITLE
An observation with respect to the beaglebone blue

### ISSUE THREAD

**Issue by manbehindthemadness** [ArduPilot/ardupilot#20325]
**IF YOU DON'T REMOVE THESE FOUR LINES, THEN YOUR REQUEST WILL BE CLOSED AUTOMATICALLY**
Questions and user problems should be directed at the forum (http://discuss.ardupilot.org)
_**Please do a careful search before opening this, there are already a lot of feature requests**_
If there was a previous discussion in the forum, link to it

## Feature request

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
After inspecting the schematics I have found something interesting.  I know that as it stands the four DC  motor controllers aren't supported; however, if we look at the schematics it would be easily possible to retask them as SSRs being that the PWM control is on the positive side of the bridge.

**Describe alternatives you've considered**
See attached wiring diagram (only the OrangeRx wiring is relevant)
In this arrangement the DSM (or any other device up to 1.2a) module can be toggled on and off by simply issuing the motor1.set(-1) and motor1.free_spin() commands respectively. It strikes be that this would be a handy additional mapping for the relay configurations as it allows control without any additional circuitry.

In short the H-Bridge can be hacked to operate as a ground switch as long as the regulated 5/3.5v rails are used as vcc instead of the second half of the bridge.

https://imgur.com/a/q3Y6xXI

**Platform**
[ * ] All
[  ] AntennaTracker
[  ] Copter
[  ] Plane
[  ] Rover
[  ] Submarine

**Additional context**
_Add any other context or screenshots about the feature request here._
